app.rb

```diff
@@ -63,11 +83,19 @@ def create_a_person
 
     case input
     when '1'
+      # rubocop:disable Layout/LineLength
       student = @create_student.new_student
-      @persons << student
+      s_struct = Struct.new(name: student.name, id: student.id, className: student.class, parent_permission: student.parent_permission, age: student.age)
```

Comment:
- [ ] It is throwing an error here when I am trying to create a new person kindly update it
![image](https://user-images.githubusercontent.com/62705947/195442268-1b49c797-01ee-4b6c-a7df-7120c35d07f3.png)